include/args.h

```diff
@@ -104,6 +104,17 @@ struct CreateMultipartUploadArgs : public ObjectArgs {
   utils::Multimap headers;
 };  // struct CreateMultipartUploadArgs
 
+struct ListMultipartUploadArgs : public BucketArgs{
+  std::string delimiter;
+  std::string encoding_type;
+  std::string key_marker;
+  unsigned int max_uploads = 1000;
+  std::string prefix;
+  std::string upload_id_marker;
+  utils::Multimap extra_headers;
+  utils::Multimap extra_query_params;
```

Comment:
`extra_headers` and `extra_query_params` are duplicates, already there in `BaseArgs`